### PR TITLE
RavenDB-21116 External Replication - block in community

### PR DESCRIPTION
**Pull request by kwiato**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-21116/External-Replication-block-in-community

### Additional description

Blocked external replication for community licenses

_Please delete below the options that are not relevant_

### Type of change

- New feature

### How risky is the change?

- Not relevant

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing by Contributor

- It has been verified by manual testing

### Testing by RavenDB QA team

- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/tasks/createOngoingTask.ts [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-import dialogViewModelBase = require("viewmodels/dialogViewModelBase"); 
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import licenseModel from "models/auth/licenseModel";
 
 class createOngoingTask extends dialogViewModelBase {
 
     view = require("views/database/tasks/createOngoingTask.html");
-
+    
+    licenseType = licenseModel.licenseStatus().Type; //TODO Get feature availability form API
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/tasks/createOngoingTask.ts:11]: there is new field to obtain that IIRC.

File: src/Raven.Studio/typescript/viewmodels/database/tasks/editExternalReplicationTask.ts [hunk 2]
```diff
@@ -11,10 +11,12 @@ import getPossibleMentorsCommand = require("commands/database/tasks/getPossibleM
 import connectionStringRavenEtlModel = require("models/database/settings/connectionStringRavenEtlModel");
 import jsonUtil = require("common/jsonUtil");
 import discoveryUrl = require("models/database/settings/discoveryUrl");
+import licenseModel from "models/auth/licenseModel";
 
 class editExternalReplicationTask extends viewModelBase {
 
     view = require("views/database/tasks/editExternalReplicationTask.html");
+    licenseType = licenseModel.licenseStatus().Type; //TODO Get feature availability form API
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/tasks/editExternalReplicationTask.ts:19]: same here

**Comment by ml054** [comment]
@kalczur 
Can you have a look at this PR and update to latest agreements?

**Comment by ml054** [comment]
@ppekrol I assume this button shouldn't be blocked, right? 

If yes - please close PR w/o merging.

**Comment by kalczur** [comment]
@ml054  In this PR, the entire view is blocked and the "Professional +" label is added, so not all of it should be removed.

**Comment by ml054** [comment]
as agreed with @ppekrol :
in 5.4 we block the button
in 6.0 we don't block button, but after clicking view is disabled + we have info in 'info hub'

**Pull request merged**